addons/redis/scripts/redis-sentinel-start-v2.sh

```diff
@@ -42,6 +42,41 @@ build_redis_sentinel_conf() {
   echo "build redis sentinel conf succeeded!"
 }
 
+check_register_sentinel_conf() {
+  if [ -f /data/sentinel/init_done.conf ]; then
+    echo "normal start"
+  else
+    echo "horizontal scaling"
+    touch /data/sentinel/init_done.conf
+    register_sentinel_conf
+  fi
+}
+
+register_sentinel_conf() {
+
+  if [[ -n "$KB_ITS_0_HOSTNAME" ]]; then
+    sentinel_pod_fqdn="$KB_ITS_0_HOSTNAME"
```

Comment:
Do not use the KB_XX_HOSTNAME environment variable. Instead, retrieve the Pod FQDN of other sentinel nodes from the current sentinel component. Explicitly define and obtain the sentinel Pod FQDN from the Vars section. Take care to handle scenarios with multiple sentinels.